tasks/main.yml

```diff
@@ -8,15 +8,19 @@
 
 - name: Gradle | apt-get update
   apt: update_cache=yes
-  
+
 - name: Gradle | Install unzip
   apt: pkg=unzip state=latest
 
 - name: Gradle | Unpack the compressed Gradle
   command: unzip /tmp/gradle-{{gradle_version}}-all.zip -d /usr/local
+  args:
+    creates: /usr/local/gradle-{{gradle_version}}
 
 - name: Grade | Rename Gradle folder
   command: mv /usr/local/gradle-{{gradle_version}} /usr/local/gradle
```

Comment:
Ok, so I guess there is a problem here, specifically this command is not re-runnable

The first time it works as expected, moving `/usr/local/gradle-{{gradle_version}}` to `/usr/local/gradle`

However once `/usr/local/gradle` exists it will move `/usr/local/gradle-{{gradle_version}}` into `/usr/local/gradle`

Change `/usr/local/gradle` to be a symlink instead?